2024/day01/day01.go

```diff
@@ -0,0 +1,47 @@
+/* Copyright © 2024 cpmachado <cpmachado@protonmail> */
+package day01
+
+import (
+	"log"
+	"math"
+	"slices"
+)
+
+// Represents a Location Id
+type LocationId int
+
+// Sorts provided slices, and computes the sum of differences
+func ListDiff(lst, other []LocationId) int {
+	if len(lst) != len(other) {
+		log.Fatal("There's an assumption of the lists being the same size\n")
+	}
+	slices.Sort(lst)
```

Comment:
Having to sort all of this, would it be interesting to parse and sort from the beginning in the parsing phase?
Using:
- Insertion Sort on the result array
- Using maybe a list
- Using a bst?